site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -124,7 +124,7 @@ public function generateHistogramData($overall_scores) {
                         // otherwise add the overall grade to array and total score possible to array (possible future use)
                         if ($ov->getTaGradedGradeable() != null && $ov->getTaGradedGradeable()->getGradedGradeable()->getSubmitter()->getRegistrationSection() != null) {
                             $histogram["bTA"] = array_merge($histogram["bTA"], [$ov->getTaGradedGradeable()->getTotalScore() + $ov->getAutoGradedGradeable()->getTotalPoints()]);
-                            $histogram["tTA"] = array_merge($histogram["tTA"], [$ov->getGradeable()->getManualGradingPoints()]);
+                            $histogram["tTA"] = array_merge($histogram["tTA"], [$ov->getTaGradedGradeable()->getTotalScore()]);
```

Comment:
What is this change doing (in the context of this PR)?

Reply:
tTA holds the histogram data for manual grades, which was not being used before this. Previously it was not getting the data for manual grades since using that definition created an empty histogram. I updated it based off of the way bTA (histogram data for overall histogram) was defined a line above since it is a combination of manual graded data and auto graded data, and the manual grading data was reflected in the histogram.